test/backup.py

```diff
@@ -371,13 +380,21 @@ def _test_backup(self, tablet_type):
     self._check_data(tablet_replica1, 1, 'replica1 tablet getting data')
 
     # backup the slave
-    utils.run_vtctl(['Backup', tablet_replica1.tablet_alias], auto_log=True)
+    alias = tablet_replica1.tablet_alias
+    logging.debug("taking backup %s",str(datetime.datetime.now()))
+    if not backup_only:
+      utils.run_vtctl(['Backup', tablet_replica1.tablet_alias], auto_log=True)
+    else:
+      self._backup_only(tablet_replica3,tablet_type='backup')
```

Comment:
Wasn't this block supposed to be deleted? I'm puzzled as to how this is passing, because there is no definition of `_backup_only` in this file.

Reply:
This is effectively a dead code path at this point. I will remove it